server/models/historical_validator_stats.ts

```diff
@@ -0,0 +1,62 @@
+import { Validators } from './../../client/scripts/views/pages/validators/index';
+import * as Sequelize from 'sequelize';
+
+export interface HistoricalValidatorStatsAttributes {
```

Comment:
please refer to questions above about each property and why they are needed for the statistics. If you can link the spec document as well that would be helpful since I notice there are a lot of extra parameters that ideally do not need to be present on these records.

Reply:
My strategy was that the evnt-handler would perform all the computation currently being done in this file .

https://github.com/hicommonwealth/commonwealth/blob/iqra.staking.%23550.%23551/client/scripts/controllers/chain/substrate/staking.ts

and event-handler would push the computed data and  fields isLatest over frontend in the DB.WHile pushing the data event-handler would mark the 'isLatest' fields as true and whenever  any record is updated the old record would be set 'isLatest' as false.
Routes would only fetch the data and directly presented on Front-end with isLatest flag true,no computations needed.Only the data fetched from db would be mapped.

The spec document is here :
https://www.notion.so/Move-to-backend-4d995500e32645e1aea7ff8a9ef6a920